command_mode.c

```diff
@@ -1353,7 +1353,9 @@ static void cmd_view(char *arg)
 
 static void cmd_push(char *arg)
 {
-	cmdline_set_text(arg);
+	if (arg) {
+		cmdline_set_text(arg);
+	}
```

Comment:
To match the rest of the code, don't put braces around an if block with a single statement in it.